lib/ph/ui/pages/main_page/main_page.dart

```diff
@@ -37,7 +37,7 @@ class PhMainPage extends HookConsumerWidget {
             ),
           const SizedBox(height: 10),
           SizedBox(
-            width: MediaQuery.of(context).size.width - 60,
+            width: 250,
```

Comment:
Have you tested this on different screen format ?

Reply:
250 pixels is really small there won't be any problems (I can't really test on little screen format)